src/components/ui/MdxInfoBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from "react-markdown/lib/react-markdown";
+
+export type MdxInfoBlockProps = {
+  type?: string;
```

Comment:
@iamnamananand996 type should be required 

```suggestion
type: string
```